data/gamerules.txt

```diff
@@ -88,3 +88,18 @@ gamerules
 	# DEFAULT: 1000 (1000 days)
 	# ALLOWABLE VALUES: any integer >= 0.
 	"depreciation max age" 1000
+	
+	# Here you can set a global minimum departure distance instead of changing all systems individually.
+	# DEFAULT: 0 (no minimum)
+	# ALLOWABLE VALUES: any integer >= 0.
+	"system departure min" 0
+	
+	# Here you can set a global minimum arrival distance instead of changing all systems individually.
+	# DEFAULT: 0 (no minimum)
+	# ALLOWABLE VALUES: any integer >= 0.
+	"system arrival min" 0
+	
+	# Here you can set a global spawn rate multiplier to change the fleet rates of all systems.
+	# DEFAULT: 1.0 (no changes)
+	# ALLOWABLE VALUES: any float >= 0.
+	"fleet multiplier" 1.0
```

Comment:
"Fleet multiplier" implies that this variable multiplies the spawn rate of fleets, such that a `"fleet multiplier"` of 2 will double the number of fleets. Either rename this variable to something else, like `"fleet period multiplier"` and update the description accordingly, or instead divide the fleet period by this number.